server/controllers/Users.js

```diff
@@ -111,6 +114,37 @@ class Users {
     }
   }
 
+  /**
+   *
+   * @name passwordResetLink
+   * @static
+   * @param {object} req express object
+   * @param {object} res express object
+   * @memberof Users
+   * @returns {JSON} JSON object
+   */
+  static async requestPasswordResetLink(req, res) {
```

Comment:
The name of the function doesn't match the name on the JSDoc. Please fix

Reply:
Done.